src/libs/Animals/TDynamicSystem.cpp

```diff
@@ -72,16 +72,16 @@ void TDynamicSystem::Draw(HDC _dc)
 //--------------------------------------------------------------------
 void TDynamicSystem::Recalculate()
 {
+    // Why does this start at 1 rather than 0?
     for (auto i = 1; i < MAX_DYNAMIC_OBJECTS; i++)
     {
-        if (attractors[i])
+        if (attractors.at(i))
```

Comment:
vector::at has worse performance than simple array access. If you know for sure that attractors' size is MAX_DYNAMIC_OBJECTS, why use vector::at ?